modules/ROOT/pages/get-started.adoc

```diff
@@ -52,7 +52,7 @@ Govern your APIs using the following steps:
 ** View summary information in the API Governance console and in validation reports.
 
 . xref:find-and-fix-conformance-issues.adoc[Find and fix conformance issues]: 
-* In API specifications in Design Center and Exchange as you design your APIs 
+* In API specifications in Anypoint Code Builder, Design Center, and Exchange as you design your APIs 
```

Comment:
Consider presenting without the replicated "In".

Reply:
It's the lesser of evils, IMHO. This isn't new text.